lib/still/compiler/collections.ex

```diff
@@ -1,30 +1,38 @@
 defmodule Still.Compiler.Collections do
   use GenServer
 
-  alias Still.Compiler.Incremental
+  alias Still.Compiler.CompilationQueue
 
   def start_link(_) do
     GenServer.start_link(__MODULE__, %{}, name: __MODULE__)
   end
 
+  def reset do
+    GenServer.call(__MODULE__, :reset)
+  end
+
   def get(collection, parent_file) do
     GenServer.call(__MODULE__, {:get, collection, parent_file})
   end
 
   def add(file) do
-    GenServer.cast(__MODULE__, {:add, file |> Map.from_struct()})
+    GenServer.call(__MODULE__, {:add, file |> Map.from_struct()}, :infinity)
```

Comment:
Ooof, that's a tough decision. Why `:infinity`?

Reply:
Because we don't want this call to expire xD if it's taking too long, then we have to fix it, but crashing is not an option, otherwise, we'll have to restart the app.

Reply:
Is there anything we can do in case the call takes too long? I'm afraid this might leave still hanging

Reply:
You're right, let's leave this for now, and we can come back to it if it's still an issue.